src/quo/components/token.cljs

```diff
@@ -0,0 +1,53 @@
+(ns quo.components.token
+  (:require [clojure.string :as string]
+            [quo.foundations.resources :as resources]
+            [react-native.core :as rn]
+            [utils.number]))
+
+(defn- size->number
+  "Remove `size-` prefix in size keywords and returns a number useful for styling."
+  [k]
+  (utils.number/parse-int (subs (name k) 5)))
+
+(defn- token-style
+  [style size]
+  (let [size-number (if (keyword? size)
+                      (size->number size)
+                      size)]
+    (assoc style
+           :width  size-number
+           :height size-number)))
+
+(defn- get-token-image*
+  [token]
+  (let [token-name (condp #(%1 %2) token
+                     string?  (keyword (string/lower-case token))
+                     keyword? (keyword (string/lower-case (name token)))
+                     nil)]
+    (resources/get-token token-name)))
```

Comment:
oh I don't think we want to load from resources like this. It's better to use the macro so we can just pull directly from the resources/tokens folder and don't need to manually update the resources.cljs file with each token that we add. 
if anything we need to delete the entries in `resources.cljs` with respect to tokens 👍

Reply:
Interesting approach. I'll add it 👍